core/src/main/scala/org/apache/spark/sql/rapids/tool/util/CacheablePropsHandler.scala

```diff
@@ -25,6 +25,15 @@ import org.apache.spark.scheduler.{SparkListenerEnvironmentUpdate, SparkListener
 import org.apache.spark.sql.rapids.tool.AppEventlogProcessException
 import org.apache.spark.util.Utils.REDACTION_REPLACEMENT_TEXT
 
+
+/**
+ * Enum to represent different spark runtimes.
+ */
+object SparkRuntime extends Enumeration {
+  type SparkRuntime = Value
+  val SPARK, SPARK_RAPIDS, PHOTON = Value
+}
+
```

Comment:
Can we add more description to that class. It is clear to me since we were discussing it offline but I guess it would be difficult for other to recognize what is the SparkRuntime.

> val SPARK, SPARK_RAPIDS, PHOTON = Value

Same for the above, letsadd comment for each one of them.

Reply:
Added more description for the class and each enum